src/app/core/http-services/item-navigation.service.ts

```diff
@@ -88,7 +88,8 @@ export interface NavMenuItem {
   currentScore?: number,
   validated?: boolean,
   canViewContent: boolean,
-  children?: NavMenuItem[] // placeholder for children when fetched (may 'hasChildren' with 'children' not set)
+  children?: NavMenuItem[], // placeholder for children when fetched (may 'hasChildren' with 'children' not set)
+  disabled: boolean,
```

Comment:
`canViewContent` does already the same ... and it is not used so it can just be renamed.

Reply:
Fixed